app/README.md

```diff
@@ -16,4 +16,9 @@ All you have to do is specify the DB (sqlite or MySQL) you want to use.
 ```
 $ cd app
 $ sh start.sh
-```
\ No newline at end of file
+```
+
+### Test
+```
+$ python -m unittest
```

Comment:
Actually we need to
1. move to `app` directory
2. run `drucker-grpc-proto/run_codegen.sh` 
3. copy `drucker-grpc-proto/protobuf/drucker_pb2.py` and `drucker-grpc-proto/protobuf/drucker_pb2_grpc.py` under `app` directory.